tests/Lykke.Service.Operations.Tests/WorkflowTests.cs

```diff
@@ -1,191 +1,256 @@
+                    EthereumHotWallet = "0x406561F72e25af10fD28b41200FA3D52badC5A21",                    
+                    FeeSettings = new Contracts.Cashout.FeeSettingsCashoutModel
                     {
-                        RestrictedCountriesIso3 = new[] {"USA"},
-                        LKK2YAssetId = "LKK2Y"
-                    },
-                    FeeSettings = new FeeSettingsModel
-                    {
-                        FeeEnabled = true,
-                        TargetClientId = "e3fa1d1e-8e7a-44e0-a666-a442bc35515c"
+                        TargetClients = new Dictionary<string, string>()
+                        { 
+                            { "Cashout", "27fe9c28-a18b-4939-8ebf-a70061fbfa05" }                            
+                        }
                     }
                 }
             };
 
-            Trace.WriteLine(JsonConvert.SerializeObject(context, Formatting.Indented));
-
+            var inputValues = JsonConvert.SerializeObject(context);
+            
             var operation = new Operation();
+            operation.Create(Guid.NewGuid(), new Guid("1a5f673a-c8a9-4cef-bcc8-3f5aa30a12bb"), OperationType.Cashout, inputValues);
 
-            var inputValues = JsonConvert.SerializeObject(context);
+            var wf = _container.ResolveNamed<OperationWorkflow>("CashoutWorkflow", new TypedParameter(typeof(Operation), operation));
+            
+            var result = wf.Run(operation);
+
+            if (result.State == WorkflowState.Corrupted || result.State == WorkflowState.Failed)
+            {
+                Trace.WriteLine(result.ToJson());
+                return;
+            }
 
-            operation.Create(Guid.NewGuid(), new Guid("27fe9c28-a18b-4939-8ebf-a70061fbfa05"), type, inputValues);                        
+            //Trace.WriteLine(JsonConvert.SerializeObject(operation, Formatting.Indented));
+            //Trace.WriteLine("--- WF result ---");
+            //Trace.WriteLine(JsonConvert.SerializeObject(result, Formatting.Indented));
+            
+            //await wfService.CompleteActivity(operation, operation.GetConfirmationActivity().ActivityId, JObject.FromObject(new { Confirmation = "111" }));
+            
+            //Trace.WriteLine(JsonConvert.SerializeObject(operation, Formatting.Indented));
 
-            var wf = container.ResolveNamed<OperationWorkflow>(workflowType, new TypedParameter(typeof(Operation), operation));            
+            await Task.Delay(300000);
 
-            var result = wf.Run(operation);
+            Assert.Equal(OperationStatus.Accepted, operation.Status);            
+        }
 
-            Trace.WriteLine(JsonConvert.SerializeObject(operation, Formatting.Indented));
-            Trace.WriteLine(JsonConvert.SerializeObject(result, Formatting.Indented));
-            Assert.Equal(WorkflowState.Complete, result.State);
-            Assert.Equal(OperationStatus.Accepted, operation.Status);
+        //[Theory]
```

Comment:
Why is this commented?

Reply:
don't remember :( it's integration tests, draft version